From 295a8fb565bd480834a2bce32141016376e00fc1 Mon Sep 17 00:00:00 2001
From: Markus Proeller <markus.proeller@pieye.org>
Date: Thu, 10 Oct 2019 19:12:51 +0200
Subject: [PATCH 577/587] configs: Add CONFIG_VIDEO_IRS1125 to the defconfigs

Adds the Infineon IRS1125 driver module to the downstream
defconfigs.

Signed-off-by: Markus Proeller <markus.proeller@pieye.org>
---
 arch/arm/configs/bcm2709_defconfig   | 1 +
 arch/arm/configs/bcm2711_defconfig   | 1 +
 arch/arm/configs/bcmrpi_defconfig    | 1 +
 arch/arm64/configs/bcm2711_defconfig | 1 +
 arch/arm64/configs/bcmrpi3_defconfig | 1 +
 5 files changed, 5 insertions(+)

diff --git a/arch/arm/configs/bcm2709_defconfig b/arch/arm/configs/bcm2709_defconfig
index d47e37387534..2b7ec829b8e6 100644
--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -902,6 +902,7 @@ CONFIG_VIDEO_TW9906=m
 CONFIG_VIDEO_IMX219=m
 CONFIG_VIDEO_OV5647=m
 CONFIG_VIDEO_OV7640=m
+CONFIG_VIDEO_IRS1125=m
 CONFIG_VIDEO_MT9V011=m
 CONFIG_DRM=m
 CONFIG_DRM_LOAD_EDID_FIRMWARE=y
diff --git a/arch/arm/configs/bcm2711_defconfig b/arch/arm/configs/bcm2711_defconfig
index 0eebb9085d28..a414adaa52bf 100644
--- a/arch/arm/configs/bcm2711_defconfig
+++ b/arch/arm/configs/bcm2711_defconfig
@@ -874,6 +874,7 @@ CONFIG_VIDEO_TW9906=m
 CONFIG_VIDEO_IMX219=m
 CONFIG_VIDEO_OV5647=m
 CONFIG_VIDEO_OV7640=m
+CONFIG_VIDEO_IRS1125=m
 CONFIG_VIDEO_MT9V011=m
 CONFIG_DRM=m
 CONFIG_DRM_LOAD_EDID_FIRMWARE=y
diff --git a/arch/arm/configs/bcmrpi_defconfig b/arch/arm/configs/bcmrpi_defconfig
index 8d81f1dd45ce..fc5c9c3c8aee 100644
--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -894,6 +894,7 @@ CONFIG_VIDEO_TW9906=m
 CONFIG_VIDEO_IMX219=m
 CONFIG_VIDEO_OV5647=m
 CONFIG_VIDEO_OV7640=m
+CONFIG_VIDEO_IRS1125=m
 CONFIG_VIDEO_MT9V011=m
 CONFIG_DRM=m
 CONFIG_DRM_LOAD_EDID_FIRMWARE=y
diff --git a/arch/arm64/configs/bcm2711_defconfig b/arch/arm64/configs/bcm2711_defconfig
index 426734caca5a..206921b8f979 100644
--- a/arch/arm64/configs/bcm2711_defconfig
+++ b/arch/arm64/configs/bcm2711_defconfig
@@ -901,6 +901,7 @@ CONFIG_VIDEO_TW9906=m
 CONFIG_VIDEO_IMX219=m
 CONFIG_VIDEO_OV5647=m
 CONFIG_VIDEO_OV7640=m
+CONFIG_VIDEO_IRS1125=m
 CONFIG_VIDEO_MT9V011=m
 CONFIG_DRM=m
 CONFIG_DRM_LOAD_EDID_FIRMWARE=y
diff --git a/arch/arm64/configs/bcmrpi3_defconfig b/arch/arm64/configs/bcmrpi3_defconfig
index ecb04d8d9106..47dde2651573 100644
--- a/arch/arm64/configs/bcmrpi3_defconfig
+++ b/arch/arm64/configs/bcmrpi3_defconfig
@@ -756,6 +756,7 @@ CONFIG_VIDEO_TW9906=m
 CONFIG_VIDEO_IMX219=m
 CONFIG_VIDEO_OV5647=m
 CONFIG_VIDEO_OV7640=m
+CONFIG_VIDEO_IRS1125=m
 CONFIG_VIDEO_MT9V011=m
 CONFIG_DRM=m
 CONFIG_DRM_LOAD_EDID_FIRMWARE=y
-- 
2.17.1

